src/action.py

```diff
@@ -216,9 +238,12 @@ def make_actor(say):
     actor.add_keyword(_('repeat after me'),
                       RepeatAfterMe(say, _('repeat after me')))
 
-    # =========================================
-    # Makers! Add your own voice commands here.
-    # =========================================
+# =========================================
+# Makers! Add your own voice commands here.
+# =========================================
+
+    actor.add_keyword(_('power off'), PowerCommand(say, 'shutdown'))
+    actor.add_keyword(_('reboot'), PowerCommand(say, 'reboot'))
```

Comment:
Yes maybe commented out so that users know they have enabled the commands. The problem is at the moment that once you've built the AIY Projects box you end up with a headless pi with ssh turned off. You can't shut it down safely without attaching a keyboard and a monitor.